### PR TITLE
Prevents ghosts from calling elevators

### PR DESCRIPTION
**Pull request by hazelrat**
Prevents ghosts from calling elevators using elevator buttons.

I assume there's way better ways to handle this. Notably, this prevents admin ghosts from using elevator buttons as well as regular ghosts. Elevator panels, like the ones in operations, have a TGUI interface that can't be opened by ghosts at all, but can be used with full functionality by admin ghosts.

Resolves https://github.com/Aurorastation/Aurora.3/issues/18330

### CODE REVIEW

**Comment by hazelrat** [comment]
!bugfix

**Comment by hazelrat** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/turbolift/turbolift_console.dm [hunk 1]
```diff
@@ -72,6 +72,14 @@
 	light_up = FALSE
 	update_icon()
 
+// Prevents ghosts from calling the lift unless they are an admin.
+
+/obj/structure/lift/button/attack_ghost(var/mob/user)
+	if(check_rights(R_ADMIN))
+		return ui_interact(user)
+	else
+		return
```
Comment (FluffyGhoster) [inline code/modules/turbolift/turbolift_console.dm:81]:
```suggestion
```

File: code/modules/turbolift/turbolift_console.dm [hunk 2]
```diff
@@ -72,6 +72,14 @@
 	light_up = FALSE
 	update_icon()
 
+// Prevents ghosts from calling the lift unless they are an admin.
+
```
Comment (FluffyGhoster) [inline code/modules/turbolift/turbolift_console.dm:76]:
```suggestion
```

File: code/modules/turbolift/turbolift_console.dm [hunk 3]
```diff
@@ -72,6 +72,10 @@
 	light_up = FALSE
 	update_icon()
 
+/obj/structure/lift/button/attack_ghost(var/mob/user)
+	if(check_rights(R_ADMIN))
```
Comment (NonQueueingMatt) [inline code/modules/turbolift/turbolift_console.dm:76]:
```suggestion
	if(check_rights(R_ADMIN, FALSE, user))
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Pull request merged**